types/prosemirror-view/index.d.ts

```diff
@@ -620,7 +624,7 @@ export interface EditorProps<S extends Schema = any> {
  * The props object given directly to the editor view supports two
  * fields that can't be used in plugins:
  */
-export interface DirectEditorProps<S extends Schema = any> extends EditorProps<S> {
+export interface DirectEditorProps<S extends Schema = any> extends EditorProps<any, S> {
```

Comment:
This is a bit subtle. The functions in `EditorProps` can be used in two ways: A Plugin can implement them (in which case the value of `T`, the plugin's state type, may have been specified), or the `EditorView` itself can keep a collection of `Plugin`s, each of which has its own, unique value for `T`. Since `DirectEditorProps` is used only by the `EditorView`, `DirectEditorProps` does not and should not have a `T` parameter, it just extends `EditorProps<any, S>`.

Reply:
Please use `unknown` instead of `any` [as recommended in the TypeScript handbook](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any).